frontends/d3ddiagex/src/main.cpp

```diff
@@ -41,6 +52,19 @@ int main(int argc, char* argv[]){
             print_version();
         }
         // TODO: Add application code here
+		std::cout << "Display Information of DirectX!\n";
+		std::cout << "Waiting........" << std::endl;
+
+		d3dcore::D3DInfo<system3d_info::GraphicContainer> d3dInfo;
+		ConsoleVisitor console_visitor;
+		d3dInfo.init();
+		d3dInfo.query_info();
+
+		auto rootContainer = d3dInfo.get_root_container();
+		rootContainer->accept(console_visitor);
+
+		std::cout << "DONE!!!!!!!!!!!!!!!!" << std::endl;
```

Comment:
Output should look short and informative and neutral, avoid using "!" unless it is critical error